core/web-assets/src/main/assets/js/apps/onms-requisitions/lib/scripts/model/RequisitionInterface.js

```diff
@@ -65,25 +68,47 @@ const RequisitionInterface = function RequisitionInterface(intf) {
   self.services = [];
 
   angular.forEach(intf['monitored-service'], function(svc) {
-    self.services.push({ name: svc['service-name'] });
+    self.services.push(new RequisitionService(svc));
   });
 
   /**
-  * @description Adds a new monitored service to the interface
-  *
-  * @name RequisitionInterface:addNewService
-  * @ngdoc method
-  * @methodOf RequisitionInterface
-  * @returns {object} the new service Object
-  */
-  self.addNewService = function() {
-    self.services.push({ name: '' });
-    return self.services.length - 1;
-  };
+   * @description The array of requisition meta-data entries
+   * @ngdoc property
+   * @name RequisitionNode#requisitionMetaData
+   * @propertyOf RequisitionNode
+   * @returns {object} The requisition meta-data entries
+   */
+  self.requisitionMetaData = [];
+
+  /**
+   * @description The array of other meta-data entries
+   * @ngdoc property
+   * @name RequisitionNode#otherMetaData
+   * @propertyOf RequisitionNode
+   * @returns {object} The other meta-data entries
+   */
+  self.otherMetaData = {};
+
+  angular.forEach(intf['meta-data'], function(entry) {
+    if (entry.context === 'requisition') {
+      self.requisitionMetaData.push({
+        'key': entry.key,
+        'value': entry.value,
+      });
+    } else {
+      if (!_.has(self.otherMetaData, entry.context)) {
```

Comment:
You know you could avoid an entire lodash import by just making this:

```
if (!self.otherMetaData.hasOwnProperty(entry.context)) {
```

The only thing useful about using `._has()` instead is if `self.otherMetaData` is undefined, but you're unconditionally assigning to it anyways.

Reply:
Yea, I know this. I was just mimicking the way it was done in the other models :sweat_smile:. Do you think it's woth changing?

Reply:
I would say it makes sense to move away from lodash if we don't need an extra dependency.  Especially if you're having to _add_ it to existing files.

Reply:
(To be clear, not saying you have to refactor all the things you're not touching in this PR or anything, just don't add new unnecessary dependencies.)

Reply:
Changed.